events/triggers/update

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+check_dependencies () {
+  for cmd in "$@"; do
+    if ! command -v $cmd >/dev/null 2>&1; then
+      echo "This script requires \"${cmd}\" to be installed"
+      exit 1
+    fi
+  done
+}
+
+check_dependencies jq wget git rsync
+
+# UMBREL_ROOT=/home/umbrel
+UMBREL_ROOT=$(dirname $(readlink -f ${BASH_SOURCE[0]}))/../..
+RELEASE=$(cat $UMBREL_ROOT/statuses/update-status.json | jq .updateTo -r)
+
+echo
+echo "======================================="
+echo "============= OTA UPDATE =============="
+echo "======================================="
+echo "========== Stage: Download ============"
+echo "======================================="
+echo
+
+# Make sure an update is not in progres
+if [[ -f "$UMBREL_ROOT/statuses/update-in-progress" ]]; then 
+    echo "An update is already in progress. Exiting now."
+    exit 2
+fi
+
+echo "Creating lock"
+touch $UMBREL_ROOT/statuses/update-in-progress
+
+# Cleanup just in case there's temp stuff lying around from previous update
+echo "Cleaning up any previous mess"
+[[ -d $UMBREL_ROOT/.umbrel-$RELEASE ]] && rm -rf $UMBREL_ROOT/.umbrel-$RELEASE
+
+# Update status file
+cat <<EOF > $UMBREL_ROOT/statuses/update-status.json
+{"state": "installing", "progress": 10, "description": "Downloading Umbrel $RELEASE", "updateTo": "$RELEASE"}
+EOF
+
+# Clone new release
+echo "Downloading Umbrel $RELEASE"
+mkdir -p $UMBREL_ROOT/.umbrel-$RELEASE
+cd $UMBREL_ROOT/.umbrel-$RELEASE
+wget -qO- "https://raw.githubusercontent.com/getumbrel/umbrel/$RELEASE/install-box.sh" | sh
```

Comment:
I don't think we should be `wget | sh`ing here.

It makes things quite unpredictable since we have no idea what is going to be in future versions if that scripts. It's convenient to give to people as easy install instructions but I don't think we should be using it as part fo the update process.

We could just wget the release tarball and extract it.

That's a lot safer, more predictable, and also makes adding signature verification in the future a lot easier.

Reply:
Actually don't worry about this for now. We can get this merged in and the I can implement the tarball install is a follow up PR and we can test deploying it OTA.

Reply:
`sounds-good-to-me.mp3`